reconcile/test/change_owners/test_change_type_diff.py

```diff
@@ -568,16 +560,20 @@ def test_bundle_change_diff_item_reorder():
         },
     )
 
-    assert not bundle_change
+    # there is a bundle change ...
+    assert bundle_change
+    # ... but it has no diffs ...
+    assert not bundle_change.diffs
+    # ... but it has different SHAs
+    assert bundle_change.old_content_sha != bundle_change.new_content_sha
```

Comment:
so we consider list order is never important, right ?

Reply:
this was an initial agreement and in most cases it is not important.
but it might be important in some cases and there was an idea lingering around to have schema metadata available to state if the order is important or not.